ntb_hw_switchtec.c

```diff
@@ -572,6 +556,37 @@ static void switchtec_ntb_check_link(struct switchtec_ntb *sndev,
 	}
 }
 
+static void check_link_status_work(struct work_struct *work)
```

Comment:
This function doesn't do anything but call another function. It's probably not needed.